Tests/test_PDB_parse_pdb_header.py

```diff
@@ -0,0 +1,50 @@
+"""Unit tests for PARTS of the parse_pdb_header module of Bio.PDB"""
```

Comment:
Missing copyright and license header. Assuming you agree with the dual licensing #898, please use:

```python
# Copyright 2017 by Bernhard Thiel.  All rights reserved.
# This file is part of the Biopython distribution and governed by your
# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
# Please see the LICENSE file that should have been included as part of this
# package.
```